contact.html

```diff
@@ -0,0 +1,138 @@
+        </header>
+    </div>
+
+
+    <div class="container content">
+        <row>
+            <div class="col-sm-12 col-md-6">
+                <div class="row"></div>
+                <div class="col-xs-6 col-sm-4"><img src="img/doctor1.jpg" alt="" class="img-responsive"></div>
+                <div class="col-xs-8 col-sm-8">
+                    <h3>Dr. Joe Smith</h3>
+                </div>
+                <div class="col-xs-12 col-sm-8">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Veritatis nesciunt ipsa aut! Nihil amet ea sequi voluptates pariatur corrupti, commodi voluptatum culpa. Laborum, cumque quod.</p>
+                </div>
+                <div class="row"></div>
+                <div class="col-xs-6 col-sm-4"><img src="img/doctor2.jpg" alt="" class="img-responsive"></div>
+                <div class="col-xs-8 col-sm-8">
+                    <h3>Dr. Jane Smith</h3>
+                </div>
+                <div class="col-xs-12 col-sm-8">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut quos voluptatem at sit perferendis modi, impedit vero error laboriosam. Sunt ut voluptate aperiam ducimus cumque?</p>
+                <div class="row">
+                    <table class="table table-striped">
+                        <thead>
+                            <tr>
+                                <th>Department</th>
+                                <th>Phone Number</th>
+                                <th>Extension</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td>Register</td>
+                                <td>834-897-0989</td>
+                                <td>x890</td>
+                            </tr>
+                            <tr>
+                                <td>Dog Toys</td>
+                                <td>834-897-0989</td>
+                                <td>x890</td>
+                            </tr>
+                            <tr>
+                                <td>Cat Stuff</td>
+                                <td>834-897-0989</td>
+                                <td>x890</td>
+                            </tr>
+                            <tr>
+                                <td>Parrot Stuff</td>
+                                <td>834-897-0989</td>
+                                <td>x890</td>
+                            </tr>
+                            <tr>
+                                <td>Vet</td>
+                                <td>834-897-0989</td>
+                                <td>x890</td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+                </div>
+            </div>
+            <div class="col-sm-12 col-md-6">
+                <div class="row">ADD IFRAME MAP HERE</div>
```

Comment:
You don't have to add this now but after merge. You should create a branch to add the map iframe.